packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/OrderSummary/template.sdd.success.tsx

```diff
@@ -182,22 +165,21 @@ const Success: React.FC<Props> = props => {
 
           <Bottom>
             {props.order.state === 'PENDING_CONFIRMATION' ||
-              (props.order.state === 'PENDING_DEPOSIT' &&
-                !props.order.paymentMethodId && (
-                  <Button
-                    data-e2e='sbSDDCancelPending'
-                    size='16px'
-                    height='48px'
-                    nature='light-red'
-                    onClick={handleCancel}
-                    style={{ marginBottom: '16px' }}
-                  >
-                    <FormattedMessage
-                      id='modals.simplebuy.summary.cancelbuy'
-                      defaultMessage='Cancel Buy'
-                    />
-                  </Button>
-                ))}
+              (props.order.state === 'PENDING_DEPOSIT' && !props.order.paymentMethodId && (
```

Comment:
maybe move `PENDING_DEPOSIT ` to enum

Reply:
this is `SBOrderStateType` which should be turned into an enum, but out the scope of this issue. I've added a TODO though.